tests/unit/trial/test_doubles.py

```
@@ -0,0 +1,272 @@
+    @staticmethod
+    def test_an_empty_reply_streams_nothing():
+        double = _coding()
+        double.replies()
+        deltas: list[str] = []
+
+        _answered(double=double, call=_call("work"), on_delta=deltas.append)
+
+        assert not deltas
+
+    @staticmethod
+    def test_a_scripted_tool_meets_the_gate_the_call_declared():
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `test_a_scripted_tool_meets_the_gate_the_call_declared`

Add return type annotation: `None`

(ANN205)